composer.json

```
@@ -2,7 +2,7 @@
     "name": "auth0/wp-auth0",
     "description": "WordPress Plugin for Auth0",
     "homepage": "https://auth0.com/wordpress",
-    "license": "GPLv2",
+    "license": "GPL-2.0-or-later",
```

```suggestion
    "license": "MIT",
```

https://github.com/auth0/wp-auth0/blob/master/LICENSE